core/src/test/scala/pureconfig/PathUtils.scala

```diff
@@ -0,0 +1,22 @@
+package pureconfig
+
+import java.nio.charset.StandardCharsets
+import java.nio.file.{ Files, Path, Paths }
+
+object PathUtils {
+
+  def createTempFile(content: String): Path = {
```

Comment:
I prefer `createTempFile` because, while it indeed writes the content of the file, it also create the file and return the `Path`. The fact that it writes is only one of the steps :).